### PR TITLE
Stdout/StderrSpec Closes #1035

### LINKED ISSUE #1035: Assess/Fix/Implement Ignored Tests in StdoutStderrWorkflowSpec
The goal here is to look at the tests in the spec, and determine
1. does the test assess something valuable
2. if no, delete it (with an explanation in the PR)
3. if yes, either fix it up so it passes, or reimplement... whatever is more expedient

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: engine/src/test/scala/cromwell/CromwellTestkitSpec.scala [hunk 1]
```diff
@@ -412,81 +411,47 @@ abstract class CromwellTestkitSpec extends TestKit(new CromwellTestkitSpec.TestW
...
+        def checkLogs(): Future[WorkflowId] = Future {
+          import better.files._
+          val (stdout, stderr) = getFirstCallLogs(workflowId)
+          Paths.get(stdout).contentAsString shouldBe expectedStdoutContent
+          Paths.get(stderr).contentAsString shouldBe expectedStderrContent
+          workflowId
         }
+
+        // Retry because we have no guarantee that all the metadata is there when the workflow finishes...
+        Await.result(Retry.withRetry(
```
Comment (mcovarr) [inline engine/src/test/scala/cromwell/CromwellTestkitSpec.scala]: @geoffjentry hopefully extractable pattern for non-cheesy eventual consistency handling, though `Retry` would need to be ~~copy/pasted~~ refactored to a common dependency

Comment (Horneth) [inline engine/src/test/scala/cromwell/CromwellTestkitSpec.scala]: The more time I spend in `CromwellTestkitSpec` the more I dislike it :(

**Comment by mcovarr** [comment]
:+1: turning on a couple of tests diminishes coverage?

**Comment by Horneth** [comment]
Yeah weird, I don't have that much trust in coverall honestly, because although it's -0.02 it's also green..

**Comment by geoffjentry** [comment]
👍

**Pull request merged**